usefulUtilities/statusIndicator/hifiScript/statusIndicator.js

```diff
@@ -161,7 +161,7 @@
 
     // Sends status to database
     var request = Script.require('request').request,
-        REQUEST_URL = Script.require(Script.resolvePath('./secrets.json')).REQUEST_URL;
+        REQUEST_URL = Script.require(Script.resolvePath('./secrets.json' + Date.now())).REQUEST_URL;
```

Comment:
```suggestion
        REQUEST_URL = Script.require(Script.resolvePath('./secrets.json?' + Date.now())).REQUEST_URL;
```

Reply:
Did you test this before submitting? :)

Reply:
No I haven't wanted CR to pass before testing in case I had to change something again.